lib/defaults/defaults.go

```diff
@@ -767,3 +769,36 @@ const (
 	// TeleportConfigVersionV2 is the teleport proxy configuration v2 version.
 	TeleportConfigVersionV2 string = "v2"
 )
+
+// Default values for TSH and TCTL commands.
```

Comment:
Nit: it's odd to see these tools written in all caps.
```suggestion
// Default values for tsh and tctl commands.
```